src/oscar/static_src/oscar/scss/fontawesome/_mixins.scss

```diff
@@ -0,0 +1,56 @@
+// Mixins
```

Comment:
Everything in `scss/fontawesome` shouldn't be committed - font awesome should be a dependency in `package.json` and compiled at build time the same as we do with Bootstrap SCSS.